### PR TITLE
CU-2nuqkfx - Set up Common codeowners

### PR DESCRIPTION
**Pull request by martyu**
CU-2nuqk3u - Set up iOS github team and codeowners
CU-2nuqk36 - Set up Android github team and codeowners

Need to set up github teams before merging.  I don't have permissions to do that.

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-2nuqk36 Set up Android github team and codeowners](https://app.clickup.com/t/2nuqk36)

**Comment by newmskywalker** [comment]
Task linked: [CU-2nuqk3u Set up iOS github team and codeowners](https://app.clickup.com/t/2nuqk3u)

**Comment by newmskywalker** [comment]
Task linked: [CU-2nuqkfx Set up Common codeowners](https://app.clickup.com/t/2nuqkfx)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# default
+* @newm-mobile/android
+
+# iOS
+/newm-mobile/iosApp @newm-mobile/ios
+/newm-mobile/shared/src/iosMain @newm-mobile/ios
+
+# Android
+/newm-mobile/android @newm-mobile/ios
+/newm-mobile/shared/androidMain @newm-mobile/ios
```
Comment (cristhianescobar) [inline .github/CODEOWNERS:10]: Should this be `@newm-mobile/android`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
```
Comment (cristhianescobar) [inline .github/CODEOWNERS]: Should this be @newm-mobile/android

**Review by cristhianescobar — CHANGES_REQUESTED** [review]

**Review by cristhianescobar — APPROVED** [review]
💯  LGTM!

**Pull request merged**